packages/altair-api/src/query-collections/dto/create-query-collection.dto.ts

```diff
@@ -17,7 +18,8 @@ export class CreateQueryCollectionDto implements ICreateQueryCollectionDto {
   name!: string;
 
   @IsOptional()
-  @ValidateNested()
+  // @ValidateNested({ each: true })
```

Comment:
disabled since the output was `{}` for some reason in the tests. Will need to revisit later.